src/main/java/com/iexec/worker/compute/post/PostComputeService.java

```diff
@@ -112,6 +131,33 @@ public PostComputeResponse runStandardPostCompute(TaskDescription taskDescriptio
         return PostComputeResponse.builder().build();
     }
 
+    Optional<ReplicateStatusCause> checkResultFilesName(String taskId, String iexecOutPath) {
+        final AtomicBoolean failed = new AtomicBoolean(false);
+        try {
+            Files.walkFileTree(Paths.get(iexecOutPath), new SimpleFileVisitor<>() {
+                @Override
+                public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) {
+                    final String fileName = file.getFileName().toString();
+                    if (fileName.length() > RESULT_FILE_NAME_MAX_LENGTH) {
+                        log.error("Too long result file name [chainTaskId:{}, file:{}]", taskId, file);
+                        failed.set(true);
+                        return FileVisitResult.TERMINATE;
+                    }
+                    return FileVisitResult.CONTINUE;
+                }
+            });
+        } catch (IOException e) {
+            log.error("Can't check result files [chainTaskId:{}]", taskId);
+            return Optional.of(POST_COMPUTE_FAILED_UNKNOWN_ISSUE);
+        }
+
+        if (failed.get()) {
+            log.error("Too long result file name [chainTaskId:{}]", taskId);
```

Comment:
Are we sure we want to log the error again ?

Reply:
Good catch! Please see https://github.com/iExecBlockchainComputing/iexec-worker/pull/538/commits/9df91b9d14c4ecc705290999995ec84dc8b74a53.